dhcpv6/dhcpv6message.go

```diff
@@ -237,20 +361,13 @@ func (m *Message) IsNetboot() bool {
 // IsOptionRequested takes an OptionCode and returns true if that option is
 // within the requested options of the DHCPv6 message.
 func (m *Message) IsOptionRequested(requested OptionCode) bool {
-	for _, optoro := range m.GetOption(OptionORO) {
-		for _, o := range optoro.(*OptRequestedOption).RequestedOptions() {
-			if o == requested {
-				return true
-			}
-		}
-	}
-	return false
+	return m.Options.RequestedOptions().Contains(requested)
```

Comment:
Same as above. If there are multiple `OptionORO` now you check only one.

Reply:
In v4, merging them is part of the RFC because of length. That's not part of v6 afaict.

RFC 8415:
>   Unless otherwise noted, each option may appear only in the options area of a DHCP message and may appear only once. 

Chapter 21.7, Option Request Option, does not appear to mention anything.

Why is the client not merging OROs?

Reply:
As above, this is an array of option codes. `Contains` is a function that checks them all.

Reply:
> Why is the client not merging OROs?

I have seen it in proprietary clients. I would still support multiple OROs.

Reply:
Yes, but `RequestedOptions` only returns the first `ORO`.

Reply:
`RequestedOptions` now merges them.

Reply:
Thanks!